### DISCUSSION TITLE
Configure MinIO to authenticate against a AzureAD which is hosted behind a proxy

### DISCUSSION THREAD

**Discussion by TRensing** [minio/minio#15952] [Q&A] [answered]
Hi,

I sucessfully configured minIO to use AzureAD as an authentication provider. Unfortunately my customer decided to make AzureAD only accessable via a proxy.

I spent a lot of time to figure out, how to configure minIO to use this proxy for authentication requests, with no result :-/ That leads me to conclude that this is not possible.

Can you confirm my fear?

Thanks a lot

Thomas

**Comment by ravindk89** [comment, marked as answer]
Can you walk us through what you have actually tried, and a little more about your environment? this isn't much to go on.

Looking at https://min.io/docs/minio/linux/operations/external-iam/configure-ad-ldap-external-identity-management.html , at what stages did you find issues?
